src/nwbinspector/utils.py

```diff
@@ -234,3 +234,22 @@ def __get_shape_helper(local_data):
     if hasattr(data, "__len__") and not isinstance(data, (str, bytes)):
         if not strict_no_data_load or isinstance(data, (list, tuple, set)):
             return __get_shape_helper(data)
+
+
+def strtobool(val: str) -> bool:
+    """
+    Convert a string representation of truth to True or False.
+
+    True values are 'y', 'yes', 't', 'true', 'on', and '1';
+    False values are 'n', 'no', 'f', 'false', 'off', and '0'.
+    Raises ValueError if 'val' is anything else.
+    """
+    if not isinstance(val, str):
+        raise TypeError(f"Invalid type of {val!r} - must be str for `strtobool`")
```

Comment:
this is where I differed from distuils' version - added type checking since seems no `mypy` type checking yet done on this codebase... so better explicit check here imho